src/tests/suite/omegaEdit.test.ts

```diff
@@ -23,30 +23,30 @@ import { Artifact, Backend } from '../../classes/artifact'
 import * as omegaEditClient from '../../omega_edit/client'
 import { unzipFile, runScript, killProcess } from '../../utils'
 import { before, after } from 'mocha'
+import * as fs from 'fs'
 
+const PROJECT_ROOT = path.join(__dirname, '../../../')
```

Comment:
Instead of having this relative link, resolve it via `path.resolve()`, which should also shorten the path names.

Reply:
@scholarsmate Was able to remove most of these. However, it needed to be kept in the `src/tests/runTest.ts`

Reply:
In another PR I have coming there will be a split out of some of these common items between test files as well

Reply:
actually this change causes windows tests to fail so we will need to keep this how it is